### DISCUSSION TITLE
outline is not working when using zsh

### DISCUSSION THREAD

**Discussion by seashell2010** [kingToolbox/WindTerm#151] [Q&A]
outline is always empty with zsh(oh-my-zsh) tab.

when using bash, it works fine.

Any config changes required to make it work?

THX.

**Comment by kingToolbox** [comment]
The scheme files define how to apply the syntax highlighting, folding, outlines, etc. The specific purpose of each file is explained [here](https://github.com/kingToolbox/WindTerm/issues/57#issuecomment-790874266). 

So if you want `zsh` to have an outline like `bash`, then you may need to write your own outline configuration. In fact, WindTerm should support the default `zsh` outline. However, since the $PS1 (command prompt and format) defined by each server may be different, it is almost impossible to write a general configuration to adapt to various scenarios. 

You can refer to the existing configuration files (`WindTerm_install_path/terminal/schemes/*/*.outline`, like `linux/linux.outline`) to write a new configuration file. If you encounter problems while writing the file, you can ask me here. Or you can execute a few more commands, and then take a screenshot or paste the result here, so that I can write the configuration for you.

**Reply by seashell2010** [reply]
Thanks.

i will try it.
